.env

```
@@ -11,3 +11,6 @@
 # Then the value for REACT_APP_WEB_API_CUSTOM_URL would be "http://localhost:9090".
 #
 #REACT_APP_WEB_API_SERVICE_URL=http://localhost:9090
+
+# Set URL of the Cate Hub instance.
+#REACT_APP_HUB_SERVER_BASE=http://localhost:9090
```

```suggestion
#REACT_APP_HUB_SERVER_BASE=http://localhost:9092
```

Cannot be same as REACT_APP_WEB_API_SERVICE_URL.